Packs/FraudWatch/Integrations/FraudWatch/FraudWatch.py

```diff
@@ -0,0 +1,854 @@
+    else:
+        raw_response = client.incident_get_by_reference(reference_id)
+
+    return CommandResults(
+        outputs_prefix='FraudWatch.Incident',
+        outputs=raw_response,
+        outputs_key_field='identifier',
+        raw_response=raw_response,
+        readable_output=tableToMarkdown("FraudWatch Incident", raw_response, removeNull=True)
+    )
+
+
+def fraud_watch_incident_forensic_get_command(client: Client, args: Dict) -> CommandResults:
+    """
+    Gets forensic data of an incident which corresponds to the given incident ID:
+    - Incident ID (Required): The ID of the incident to have its forensic data retrieved.
+
+    Known errors that causes error to be returned by FraudWatch service:
+    - Unknown incident id.
+
+    Args:
+        client (Client): FraudWatch client to perform the API calls.
+        args (Dict): Demisto arguments.
+
+    Returns:
+        CommandResults.
+    """
+    incident_id = args.get('incident_id')
+
+    raw_response = client.incident_forensic_get(incident_id)
+    outputs = remove_empty_elements(raw_response)
+    if outputs:
+        outputs['identifier'] = incident_id
+
+    return CommandResults(
+        outputs_prefix='FraudWatch.IncidentForensicData',
+        outputs=outputs,
+        outputs_key_field='identifier',
+        raw_response=raw_response,
+        readable_output=tableToMarkdown("FraudWatch Incident Forensic Data", outputs,
+                                        removeNull=True)
+    )
+
+
+def fraud_watch_incident_contact_emails_list_command(client: Client, args: Dict) -> CommandResults:
+    """
+    Provides contact emails for the incident which corresponds to the given incident ID:
+    - Incident ID (Required): The ID of the incident to have its email contacts data retrieved.
+    - Limit: Total number of contact emails in a page. The default limit is 20 and the maximum number is 200.
+    - Page: Retrieve contact emails by the given page number.
+
+    Known errors that causes error to be returned by FraudWatch service:
+    - Unknown incident id.
+    - Page index out of bounds.
+
+    Args:
+        client (Client): FraudWatch client to perform the API calls.
+        args (Dict): Demisto arguments.
+
+    Returns:
+        CommandResults.
+    """
+    incident_id = args.get('incident_id')
+    page = get_and_validate_positive_int_argument(args, 'page')
+    limit = get_and_validate_positive_int_argument(args, 'limit')
+
+    try:
+        raw_response = client.incident_contact_emails_list(incident_id, page, limit)
+    except DemistoException as e:
+        if 'Contact email not found' in str(e):
+            page_error_msg = f'''Make sure page index: {page} is within bounds.''' if page else ''
+            unknown_incident_msg = f'''Make sure incident id: {incident_id} is correct.'''
+            raise DemistoException(
+                f'''Error occurred in command 'fraudwatch-incident-contact-emails-list'. {page_error_msg}'''
```

Comment:
No need for command prefix, it will appear under the war-room entry
```suggestion
                f'''Error occurred. {page_error_msg}'''
```